src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import static core.basesyntax.MyHashMap.Node.hash;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_LENGTH = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] hashTable = new Node[DEFAULT_LENGTH];
+    private int size;
+    private int arrayLength;
+
+    static class Node<K, V> implements Entry<K, V> {
```

Comment:
```suggestion
    private static class Node<K, V> implements Entry<K, V> {
```
should be after public methods